packages/gatsby-theme-bulmaio/src/components/footer/Footer.spec.tsx

```diff
@@ -5,11 +5,12 @@ import 'jest-dom/extend-expect';
 import Footer, { FooterProps } from './Footer';
 
 export const DUMMY_FOOTER: FooterProps = {
-  copyright: 'copyright1'
+      copyright: 'copyright2'
```

Comment:
FYI @pauleveritt when you review, I will update formatting before merge so please ignore

Reply:
🎉